### ISSUE TITLE
How do you use the custom data property?

### ISSUE THREAD

**Issue by kcoderhtml** [vriteio/vrite#55]
```ts
async function createContentPiece(title: string, description: string, body: string, scrappyId: string, date: any) {
    const contentPiece = await vrite.contentPieces.create({
        contentGroupId: process.env.VRITE_CONTENT_GROUP_ID || '',
        title: title,
        description: description,
        content: body,
        tags: [],
        customData: { 
            id: scrappyId,
         },
        members: ["64974cb888e8beebeb2c925f"],
        date: date,
    });
    return contentPiece;
};
```
In my code, no matter what arrangement I try with the custom data field nothing is showing up in the editor; also no errors are being logged either.

**Comment by areknawo** [comment]
Seems like a bug. Will check.

**Comment by areknawo** [comment]
Should be fixed (server-side) now in v0.4.0.
